centaur/src/main/resources/standardTestCases/drs_basename.test

```diff
@@ -0,0 +1,22 @@
+name: drs_basename
+testFormat: WorkflowSuccess
+backends: ["papi-v2-usa"]
+skipDescribeEndpointValidation: true
+
+files {
+  workflow: drs_tests/drs_basename.wdl
+  options-dir: "Error: BA-6546 The environment variable CROMWELL_BUILD_RESOURCES_DIRECTORY must be set/export pointing to a valid path such as '${YOUR_CROMWELL_DIR}/target/ci/resources'"
```

Comment:
I'm confused about this options-dir (line 8).

Reply:
Are you just using it to add a comment, since it is overwritten in the next step?

Reply:
If I understand this correctly (I copy/pasted this format from another file), the file is interpreted in top-down order. This line sets an easy-to-debug wrong value, and the next line overrides it iff the environment is correctly set up.

I think this came about because if you forget to set up your environment variables properly then working out which value you were missing was incredibly difficult.

Reply:
Got it, thanks!